Resources/doc/index.md

```diff
@@ -141,6 +141,22 @@ be omitted from the DSNs. Also make sure to use the sentinel port number
 (26379 by default) in the DSNs, and not the default Redis port.
 You can find more information about this on [Configuring Sentinel](https://redis.io/topics/sentinel#configuring-sentinel).
 
+A setup using `RedisCluster` from `phpredis`  could look like this:
+
+``` yaml
+snc_redis:
+    clients:
+        default:
+            type: phpredis
+            alias: default
+            dsn: ["redis://localhost/0"]
+            options:
+                profile: 4.1
+                cluster: "phpredis-cluster"
```

Comment:
This value can be anything but `null` or `false` right?

If so, I would suggest to validate this value to `true` or `false` or `null` in the extension, what do you think?